main.sh

```diff
@@ -26,11 +26,19 @@ vus=$6
 : "${vus:=10}"
 duration=$7
 : "${duration:="5m"}"
+minio_url=$8
+: "${minio_url:="http://localhost:9000"}"
```

Comment:
```suggestion
: "${minio_url:="localhost:9000"}"
```
@trueleo This should fix https://github.com/parseablehq/quest/pull/47#issuecomment-1862669882.